angular-route/templates/_.client.route.js

```diff
@@ -0,0 +1,5 @@
+$stateProvider.
+  state('<%= slugifiedName %>', {
+    url: '/<%= slugifiedRoutePath %>',
+    templateUrl: 'modules/<%= slugifiedModuleName %>/client/views/<%= slugifiedViewName %>.client.view.html'
+  }).
```

Comment:
Again, why newline in a partial, that is used to *replace* a
`$stateProvider.` string in original file?
Newline in the end would just ruin the idea.

2015-12-21 1:38 GMT+03:00 Ilan Biala <notifications@github.com>:

> In angular-route/templates/_.client.route.js
> <https://github.com/meanjs/generator-meanjs/pull/162#discussion_r48109358>
> :
>
> > @@ -0,0 +1,5 @@
> > +$stateProvider.
> > +  state('<%= slugifiedName %>', {
> > +    url: '/<%= slugifiedRoutePath %>',
> > +    templateUrl: 'modules/<%= slugifiedModuleName %>/client/views/<%= slugifiedViewName %>.client.view.html'
> > +  }).
>
> newline
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/meanjs/generator-meanjs/pull/162/files#r48109358>.
>